src/map/clif.c

```diff
@@ -1065,6 +1065,7 @@ void clif_set_unit_idle(struct block_list* bl, struct map_session_data *tsd, enu
 #endif
 #if PACKETVER >= 20150513
 	p.body = vd->body_style;
+	memcpy(p.name, status->get_name(bl), NAME_LENGTH);
```

Comment:
what about move "memcpy(p.name, status->get_name(bl), NAME_LENGTH);" into separate function?
This will allow plugins for example translate name or do any other things with it.